dali/python/nvidia/dali/ops.py

```diff
@@ -219,6 +231,13 @@ def __init__(self, **kwargs):
                 self._device = op_device
             self._spec.AddArg("device", self._device)
 
+            if "preserve" in kwargs.keys():
+                self._preserve = kwargs["preserve"]
+            else:
+                self._preserve = False
+            self._spec.AddArg("preserve", self._preserve)
+            self._preserve = self._preserve or self._schema.IsNoPrune()
```

Comment:
Do we still need this?

Reply:
I'm thinking how we can have one implementation for deciding the preserve flag based on the argument and IsNoPrune without duplicating the logic in C++ and Python, but currently I don't have a good idea for it.